File: Tareas/Unidad2/a2253332351_practica02/18_NominaSemanal.cpp
#include <iostream>
using namespace std;

int main() {
    double sueldoPorHora;
    double horasTrabajadas;
    double pagoTotal;

    cout << "Ingrese el sueldo por hora: ";
    cin >> sueldoPorHora;
    cout << "Ingrese las horas trabajadas: ";
    cin >> horasTrabajadas;

    pagoTotal = sueldoPorHora * horasTrabajadas;

    cout << "El pago total de la semana es: " << pagoTotal << endl;

    return 0;
}

File: Tareas/Unidad2/a2253332351_practica02/18_NominaSemanal.java
import java.util.Scanner;

public class NominaSemanal {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double sueldoPorHora, horasTrabajadas, pagoTotal;

           System.out.println("Ingrese el sueldo por hora: ");
           sueldoPorHora = entrada.nextDouble();
           System.out.println("Ingrese las horas trabajadas: ");
           horasTrabajadas = entrada.nextDouble();

           pagoTotal = sueldoPorHora * horasTrabajadas;

           System.out.println("El pago total de la semana es: " + pagoTotal);
     }
}
